src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -418,7 +420,7 @@ public void testDestroyCacheWithWrongFileNameLength() throws Exception {
     }
 
     @Test
-    public void testCacheFile() throws Exception {
+    public void testCacheFileWithoutExternalResources() throws Exception {
```

Comment:
> we test caching functionality for the check which does not hold external resources.

So this is the same as `testFilterWhichDoesNotImplementExternalResourceHolderInterface` but that is testing it for a filter and this one is testing it with a check?